app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -132,6 +132,8 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
   ) async {
     final String labelName = config.waitingForTreeToGoGreenLabelName;
 
+    log.fine('Querying GitHub\'s GraphQL API...');
```

Comment:
nit: put the query in here and the result in the log below

Reply:
Thanks! I added a log for the result.data.

I'm skipping before as `QueryOptions` from package:graphql doesn't offer an override for toString() or toJson() so only the type would be printed. Additionally, the graphql query is mostly const (except for the variables inserted)